src/components/App.tsx

```diff
@@ -881,6 +884,14 @@ export default class App extends React.Component<any, AppState> {
     });
   }
 
+  onFileSelected = (e: File[]) => {
```

Comment:
Why os this here while the drop zone is not part of this component? I find it a bit strange...

Reply:
I will try to move this to AppToolbar, and pass setLocalPMTiles as a prop to AppToolbar (which handles onFileSelected then calls this.props.setlocalPMTiles). Is this what you have in mind?